frontend/src/app/components/filters/filter-integer-value/filter-integer-value.component.ts

```diff
@@ -34,6 +34,7 @@ import {I18nService} from 'core-app/modules/common/i18n/i18n.service';
 import {DebouncedEventEmitter} from 'core-components/angular/debounced-event-emitter';
 import {UntilDestroyedMixin} from "core-app/helpers/angular/until-destroyed.mixin";
 import {componentDestroyed} from "@w11k/ngx-componentdestroyed";
+import {SchemaCacheService} from "core-components/schemas/schema-cache.service";
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/0bd366590d8701974caa82d7be5aab85c106b58a/frontend/src/app/components/filters/filter-integer-value/filter-integer-value.component.ts#L38"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>